ols/src/prompts/prompts.py

```diff
@@ -15,9 +15,20 @@
 # With PromptTemplate, we have more control how we want to structure the prompt.
 
 QUERY_SYSTEM_INSTRUCTION = """
-You are an assistant for question-answering tasks \
-related to the openshift and kubernetes container orchestration platforms.
-
+You are OpenShift Lightspeed - an intelligent assistant for question-answering tasks \
+related to the OpenShift container orchestration platform.
+
+Here are your instructions:
+You are OpenShift Lightspeed, an intelligent assistant and expert on all things OpenShift. \
```

Comment:
this seems redundant to line 18?  Do we need both of these?  Can we combine them somehow?  Minimizes the system instruction length is important from a context-window perspective and response latency perspective.

Reply:
I'll drop this and do a spot check.

Reply:
If I dedupe lines 22 and 18, the LLM doesn't know what "OpenShift Lightspeed" is anymore. Instead of 

```
Q: What is OpenShift Lightspeed?

A: OpenShift Lightspeed is the identity I embody as an intelligent assistant...
```

it starts saying

```
Q: What is OpenShift Lightspeed?

A: OpenShift Lightspeed is not an official feature or component of OpenShift. It appears there might be a 
misunderstanding or confusion in the question. If you have any questions about OpenShift, such as its 
features, functionalities, or how to use it for container orchestration and management, feel free to ask!
```

Reply:
ok, just leave it i guess.  but you might consider adding a test case for this specific behavior to ensure we don't regress on being able to answer this question in the future.

for that matter you should consider tests for the other behaviors you're trying to enforce w/ these instructions, so that in the future if we revisit the instructions we have tests that ensure we maintain the desired behavior (they may have to be updated to accomodate changes in response, but at least having the tests makes it clear to future contributors what the expected behavior is that needs to be maintained)